SWLOR.Game.Server/Feature/PerkDefinition/ForcePerkDefinition.cs

```diff
@@ -450,23 +448,21 @@ private void ForceLightning()
 
         private void ForceBody()
         {
-            _builder.Create(PerkCategoryType.ForceDark, PerkType.ForceBody)
+            _builder.Create(PerkCategoryType.ForceUniversal, PerkType.ForceBody)
                 .Name("Force Body")
 
                 .AddPerkLevel()
-                .Description("Converts 25% of the user's HP into FP.")
+                .Description("Grants Force Point regeneration based on your Willpower Score, at the cost of your Vitality.")
```

Comment:
Same comment here about Force Points. Calling it FP will make it consistent across the board.